spot_driver_cpp/test/src/kinematic/test_kinematic_service.cpp

```diff
@@ -0,0 +1,7 @@
+// Copyright (c) 2023 Boston Dynamics AI Institute LLC. All rights reserved.
+
+#include <gmock/gmock.h>
+
+#include <spot_driver_cpp/api/kinematic_api.hpp>
+
+namespace spot_ros2::kinematic::test {}  // namespace spot_ros2::kinematic::test
```

Comment:
Is this file still WIP?

Reply:
This is done now. It is a very basic test.